src/wp-content/themes/twentytwelve/functions.php

```diff
@@ -201,11 +204,27 @@ function twentytwelve_scripts_styles() {
 	// Adds JavaScript for handling the navigation menu hide-and-show behavior.
 	wp_enqueue_script( 'twentytwelve-navigation', get_template_directory_uri() . '/js/navigation.js', array( 'jquery' ), '20141205', true );
 
-	$font_url = twentytwelve_get_font_url();
-	if ( ! empty( $font_url ) ) {
-		wp_enqueue_style( 'twentytwelve-fonts', esc_url_raw( $font_url ), array(), null );
+	/**
+	 * The dependencies for our font stylesheet.
+	 *
+	 * We register our sub-fonts as dependencies for the main font stylesheet. This way, we can
```

Comment:
I did not understand what  "sub-fonts" are.   Is there perhaps a more user friendly term?

Reply:
I understand the issue, but I also lack a better term right now. Maybe 'font variants' or something similar?

Reply:
I think font variants would work.

Reply:
renamed sub-fonts to font variants